io.enpass.Enpass.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+Name=Enpass
+GenericName=Enpass Password manager
+Icon=io.enpass.Enpass
+Terminal=false
+Exec=/app/bin/enpass %U
```

Comment:
```suggestion
Exec=enpass %U
```

Reply:
Thank you.

I've removed the absolute path :heavy_check_mark: